[
    {
        "content": "<p>I'm trying to run stage 1 test suite and it is failing at tidy check.</p>\n<p><code>./x.py test tidy</code> log: <a href=\"http://csclub.uwaterloo.ca/~a52arora/tidy-error\">http://csclub.uwaterloo.ca/~a52arora/tidy-error</a></p>\n<p>Is there a way to bypass tidy when running tests? I checked <code>./x.py test -h</code> but couldn't find anything relevant. </p>\n<p>I just rebased off of master, ran <code>./x.py fmt</code>, also haven't made any changes to the backtrace crate</p>",
        "id": 205058790,
        "sender_full_name": "Aman Arora",
        "timestamp": 1595782073
    },
    {
        "content": "<p>Actually looks like backtrace was removed as a submodule: <a href=\"https://github.com/rust-lang/rust/commit/cc4f547cf4a6bbeb7260077aca26be88644cb019\">https://github.com/rust-lang/rust/commit/cc4f547cf4a6bbeb7260077aca26be88644cb019</a></p>\n<p>Should x.py be removing directories that correspond to submodules that were removed? Or should test not run tidy check on untracked files?</p>",
        "id": 205059039,
        "sender_full_name": "Aman Arora",
        "timestamp": 1595782477
    },
    {
        "content": "<p>IIRC other commends ignore untracked directories and files</p>",
        "id": 205062765,
        "sender_full_name": "Poliorcetics",
        "timestamp": 1595789210
    },
    {
        "content": "<p>If so, maybe this is a bug in tidy</p>",
        "id": 205062772,
        "sender_full_name": "Poliorcetics",
        "timestamp": 1595789227
    }
]